__tests__/models.test.js

```diff
@@ -0,0 +1,17 @@
+const { selectCommentCountByArticleId } = require('../models/comments-model.js');
+
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("Comments model", () => {
```

Comment:
you don't need to test your models separately - they are being tested in your integration tests so this will save you a job!